### DISCUSSION TITLE
Pinging users/roles does not work with deferred replies

### DISCUSSION THREAD

**Discussion by chrisdlim** [discordjs/discord.js#9483] [Q&A] [answered]
### Which package is this bug report for?

discord.js

### Issue description

I have a command that creates a temporary queue for people to join but I want it to expire after X minutes (longer than the 15min webhook token expiration).

To do this, I defer the reply and then use a setTimeout to expire the queue after X minutes. It works fine but because of the deferred reply, the command can't ping users or roles when it's meant to. Is there a workaround for this? I'm not sure why a deferred reply isn't able to ping roles/users.

1. Send bot command
2. Defer reply
3. Return a reply with `roleMention` or `userMention`
4. Pings are never sent

### Code sample

```typescript
await interaction.deferReply();
await interaction.editReply({
  content: roleMention('roleId123456'),
  allowedMentions: { parse: ['roles'] },
});
```

### Versions

- discord.js@14.8.0
- typescript@4.9.5

### Issue priority

Low (slightly annoying)

### Which partials do you have configured?

Not applicable

### Which gateway intents are you subscribing to?

Guilds, GuildMembers, GuildMessages, MessageContent

**Comment by monbrey** [comment, marked as answer]
I think this is a standard Discord thing - message edits have never sent out new notifications. You might need to finish the defer, *then* use a followUp to ensure a new message is created and sends notifications.

**Reply by chrisdlim** [reply]
I'll give that a try! Thanks! I didn't know that "finishing a defer" was a thing. 

I did a lot of searching for "pinging with deferred replies" and never saw that as a possibility.

**Reply by chrisdlim** [reply]
@monbrey could you provide an example on how to "finish the defer"? 

I tried the following:
```
await interaction.deferReply();
await interaction.editReply({
  content: 'some fake content',
  allowedMentions: { parse: ['roles'] },
});
await interaction.followUp(roleMention('roleId123456'));
```
but it still didn't ping the role.

**Reply by Qjuh** [reply]
You might need to explicitly pass `allowedMentions` for a followUp to ping (and make sure the bot has permissions to mention that role)

**Reply by chrisdlim** [reply]
Ah, minor detail missed on my part. That worked. Thanks a lot!
